src/nls/nl/strings.js

```diff
@@ -103,63 +129,126 @@ define({
+    "FIND_TITLE_SUMMARY"                : "&mdash; {0} {1} {2} in {3}",
 
-    "FIND_IN_FILES_TITLE_PART1"         : "\"",
-    "FIND_IN_FILES_TITLE_PART2"         : "\" gevonden",
-    "FIND_IN_FILES_TITLE_PART3"         : "&mdash; {0} {1} {2} in {3} {4}",
+    //Find in Files
+    "FIND_NUM_FILES"                    : "{0} {1}",
     "FIND_IN_FILES_SCOPED"              : "in <span class='dialog-filename'>{0}</span>",
-    "FIND_IN_FILES_NO_SCOPE"            : "in project",
+    "FIND_IN_FILES_NO_SCOPE"            : "in project",    
+    "FIND_IN_FILES_ZERO_FILES"          : "Filter sluit alle bestanden uit {0}",
     "FIND_IN_FILES_FILE"                : "bestand",
     "FIND_IN_FILES_FILES"               : "bestanden",
     "FIND_IN_FILES_MATCH"               : "overeenkomst",
     "FIND_IN_FILES_MATCHES"             : "overeenkomsten",
     "FIND_IN_FILES_MORE_THAN"           : "Meer dan ",
     "FIND_IN_FILES_PAGING"              : "{0}&mdash;{1}",
-    "FIND_IN_FILES_FILE_PATH"           : "<span class='dialog-filename'>{0}</span> {2} <span class='dialog-path'>{1}</span>", // We shoudl use normal dashes on Windows instead of em dash eventually
-    "ERROR_FETCHING_UPDATE_INFO_TITLE"  : "Probleem bij het ophalen van update informatie",
-    "ERROR_FETCHING_UPDATE_INFO_MSG"    : "Er is een fout opgetreden bij het ophalen van de laatste update informatie van de server. Zorg ervoor dat je verbonden bent met het internet en probeer opnieuw.",
-
+    "FIND_IN_FILES_FILE_PATH"           : "<span class='dialog-filename'>{0}</span> {2} <span class='dialog-path'>{1}</span>", // We should use normal dashes on Windows instead of em dash eventually    
+    "FIND_IN_FILES_EXPAND_COLLAPSE"     : "Klik Ctrl/Cmd om alles in/uit te vouwen",
+    "REPLACE_IN_FILES_ERRORS_TITLE"     : "Vervang fouten",
+    "REPLACE_IN_FILES_ERRORS"           : "De volgende bestanden zijn niet aangepast omdat deze na het zoeken zijn aangepast of niet bewerkt kunnen worden.",    
+    
+    "ERROR_FETCHING_UPDATE_INFO_TITLE"  : "Probleem bij het ophalen van update-informatie",
+    "ERROR_FETCHING_UPDATE_INFO_MSG"    : "Er is een fout opgetreden bij het ophalen van de laatste update-informatie van de server. Zorg ervoor dat je verbonden bent met het internet en probeer opnieuw.",    
+    
+    // File exclusion filters
+    "NEW_FILE_FILTER"                   : "Nieuwe set uitgesloten bestanden\u2026",
+    "CLEAR_FILE_FILTER"                 : "Sluit geen bestanden uit",
+    "NO_FILE_FILTER"                    : "Geen uitgesloten bestanden",
+    "EXCLUDE_FILE_FILTER"               : "Sluit {0} uit",
+    "EDIT_FILE_FILTER"                  : "Bewerk\u2026",
+    "FILE_FILTER_DIALOG"                : "Bewerk set uitgesloten bestanden",
+    "FILE_FILTER_INSTRUCTIONS"          : "Sluit bestanden en mappen uit met de volgende stringen/substringen of <a href='{0}' title='{0}'>wildcards</a>. Plaats elke string op een nieuwe regel.",
```

Comment:
'stringen/substringen' are no Dutch words. Perhaps 'teksten/deelteksten' is a better alternative.